os/Kconfig

```diff
@@ -315,12 +315,8 @@ config MOTOROLA_SREC
 		and should not be selected if you are not using the GNU toolchain.
 
 config RAW_BINARY
-	bool "Raw binary format"
-	default n
-	---help---
-		Create the tinyara.bin in the raw binary format that is used with many
-		different loaders using the GNU objcopy program.  This option
-		should not be selected if you are not using the GNU toolchain.
```

Comment:
I think we can keep the description except last sentence. And we can add some description that this is mandatory on TizenRT.